app/views/layouts/_alerts.html.erb

```diff
@@ -1,2 +1,2 @@
-<p class="notice" ><%= notice %> </p>
-<p class="alert" ><%= alert %> </p>
\ No newline at end of file
+<p class="alert-success" style="text-align: center;" ><%= notice %> </p>
+<p class="alert-danger"style="text-align: center;" ><%= alert %> </p>
```

Comment:
Please add a new line at end of this file
Also, try to use bootstrap alerts over here.